### PR TITLE
✨ [Story devtools] Device details

### PR DESCRIPTION
**Pull request by mszylkowski**
Added hardware details and UI to the devices on the preview tab.

Androids:
image
iPhones:
image
Tablet and desktop:
image

### CODE REVIEW

File: extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css [hunk 1]
```diff
@@ -50,7 +50,8 @@
   border-radius: 30px !important;
   background: var(--i-amphtml-story-dev-tools-gray-2) !important;
   overflow: hidden !important;
-  padding-top: 40px !important;
+  display: flex;
```
Comment (gmajoulet) [inline extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css:53]: !important

**Review by gmajoulet — APPROVED** [review]

**Comment by processprocess** [comment]
I feel like the date/time/battery should be abstracted to match the rest of the fake UI. 
It's bright, busy and distracting from the intention of this. 
I strongly feel it's not necessary to have this level of detail for fake UI.
 
I'll default to @hongcatlover though.

Screen Shot 2021-01-21 at 11 38 59 AM

**Comment by mszylkowski** [comment]
@processprocess it's a good point and I think we talked about having abstract icons there, I mocked the abstract icons but it's not easy to divert attention from them. With Hong we reached the conclusion that having unfamiliar shapes are more distracting than having more realistic UI mocks. We couldn't find a good way of simplifying the UI so that it doesn't attract attention, when users see something unusual for the first time, they will look at it more than if it was regular.

Sth to take into account is: if statusbar icons are too distracting, then the mobile operating systems wouldn't have them as such. So in our tool, users should pay the same attention to them as they would if they tried the stories on their phones, and changing it will attract more attention because of unfamiliarity.

image

**Comment by processprocess** [comment]
>> Sth to take into account is: if statusbar icons are too distracting, then the mobile operating systems wouldn't have them as such. 

Status icons have a use in mobile operating systems. They have no use in an abstraction of a UI.

**Comment by processprocess** [comment]
Spoke with @hongcatlover and we decided on this:
Screen Shot 2021-01-21 at 12 27 43 PM
The bright is currently putting it visually on the same playing field as the story.
Having the color match the fake UI helps it blend and match with its intention.

**Comment by mszylkowski** [comment]
Awesome, will update designs!

**Comment by mszylkowski** [comment]
End result is this, sending the commit RN:
image
image

**Review by processprocess — APPROVED** [review]

**Pull request merged**